src/composables/tokensList.ts

```diff
@@ -1,18 +1,24 @@
 import BigNumber from 'bignumber.js';
-import { computed, Ref } from 'vue';
+import {
+  computed,
+  Ref,
+} from 'vue';
```

Comment:
This change gives no value in my opinion but makes the code more prone to conflicts.